### DISCUSSION TITLE
New design for GTK4 applications?

### DISCUSSION THREAD

**Discussion by LinuxLlama** [ubuntu/yaru#2906] [Q&A] [answered]
After compiling the latest commits, I noticed a few GTK4 applications look different. Are we to expect a Yaru refresh or is this just experimental? Either way, I like what I'm seeing and wanted to share my gratitude.

**Light theme preview** (I actually like this a lot!)
![Screenshot from 2021-06-12 09-05-13](https://user-images.githubusercontent.com/29968331/121782330-d9b75580-cb5d-11eb-8927-197f049cc120.png)

**Mixed theme preview** (The flat buttons look really nice. I like how the different corner radius's visually connect them.)
![Screenshot from 2021-06-12 09-05-42](https://user-images.githubusercontent.com/29968331/121782335-db811900-cb5d-11eb-8b5c-38ca3931b9be.png)

**Dark theme preview** (Forces rounded corners at the bottom? Normally, the extension app doesn't have rounded corners.)
![Screenshot from 2021-06-12 09-06-20](https://user-images.githubusercontent.com/29968331/121782339-dcb24600-cb5d-11eb-8145-230081db7c14.png)

**Comment by Feichtmeier** [comment, marked as answer]
Yes we now symlink 95% of the theme to the upstream gtk4 libadwaita scss and only change our color palette, the orange close dot and yet the third version with the light theme and the inverted headerbar.
The plan looks like we won't change this currently because changes are happening so rapidely that we will just keep up with them - currently nothing more is planed, we'll keep the community updated
relevant PRs https://github.com/ubuntu/yaru/pull/2901 and https://github.com/ubuntu/yaru/pull/2902

**Reply by LinuxLlama** [reply]
Awesome, thanks for the answer!

**Reply by UltraBurstXD** [reply]
GTK 4 looks pretty nice!

**Comment by Jupi007** [comment]
> Forces rounded corners at the bottom? Normally, the extension app doesn't have rounded corners

Well, Gtk4 itself hasn't got rounded corner at bottom but Libadwaita so. This app doesn't use Libadwaita at all, but as Yaru being based on it, every Gtk4 app got rounded corner at bottom.

**Reply by neffscape** [reply]
Hi guys, thank you for your amazing work and for the awesomeness of the achieved results. Given that libadwaita 1.0 and GNOME 42 have finally been released, I would like to know what are your plans for the future. We know that a new recoloring API is in development, but nobody ever talks about changing default font. Currently I really like the ubuntu font that is used in window titles, gnome shell and other elements, but Libadwaita + GTK4 are forcing Cantarell, and its bold variant is often overused. I think this not just an aesthetics problem: is this issue actually being discussed by the team that is working with upstream?

**Reply by schuelermine** [reply]
@neffscape maybe make a separate Discussion about that
